app-emulation/docker-credential-helpers/docker-credential-helpers-0.6.3.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 2001-2018 Gentoo Foundation
```

Comment:
You should use repoman to commit so it would fix you the right copyright header AND propose a good git summary line. Please update them, 
https://www.gentoo.org/glep/glep-0066.html#commit-messages

looks good otherwise!